dissect/target/plugin.py

```diff
@@ -870,6 +870,12 @@ def find_plugin_functions(target: Target, patterns: str, compatibility: bool = F
     functions, rootset = plugin_function_index(target)
 
     for pattern in patterns.split(","):
+        # OS is used as a header in the -l output of CLI
+        # this is not a valid plugin pattern but let's be friendly to the user and
+        # give all OS properties as records (osinfo is a little wrapper) (users are excluded because those are records)
+        if pattern.startswith("OS"):
```

Comment:
Perhaps consider also allowing lowercase `os`?

Reply:
Not sure if that is a good idea. OS does not exist but "os" does. So this interferes with existing functionality.

Reply:
Is this not a very weird edge case to support in that case? I think the `osinfo` is a good idea, just wondering if it's good to really put a hard `if EDGE_CASE` for one specific header in some documentation, if the intended behavior between `OS` and `os` are different. `OS` should behave similar to `os` in that case, in my opinion, as casing shouldn't have this big of a difference.

Reply:
Then we should just lower case all patterns (or do case insensitive match)? And keep `osinfo` available as its own `os.info` plugin or something like that.

Reply:
Ok adjusted